### ISSUE TITLE
Package request: Ulauncher

### ISSUE THREAD

**Issue by rowbawts** [void-linux/void-packages#5532]
https://ulauncher.io/

Already has packages for .deb, .rpm, and is in aur. Not sure if any can make the process any easier.

GitHub - https://github.com/Ulauncher/Ulauncher/

**Comment by steinex** [comment]
```
# Template file for 'ulauncher'
pkgname=ulauncher
version=4.3.1.r4
revision=1
wrksrc=${pkgname}
build_style=python2-module
pycompile_module="ulauncher"
hostmakedepends="python-distutils-extra intltool"
depends="libappindicator libkeybinder3 python-Levenshtein python-dbus
 python-inotify python-sqlite python-websocket-client python-xdg webkit2gtk"
short_desc="Linux Application Launcher"
maintainer=""
license="GPL-3.0-or-later"
homepage="https://ulauncher.io/"
distfiles="https://github.com/Ulauncher/Ulauncher/releases/download/${version}/${pkgname}_${version}.tar.gz"
checksum=695d874cdfdf9f0009278ae0c060964bb5de13b4341eacad1fca97bc18ef3cca
```

Installs but python-appindicator needs to be packaged first. Feel free to take.

**Comment by paualberto** [comment]
Can confirm that the template works, and then this happens

```
  from gi.repository import Gio, Gtk, WebKit2, GLib
Traceback (most recent call last):
  File "/usr/bin/ulauncher", line 27, in <module>
    from ulauncher.main import main
  File "/usr/lib/python2.7/site-packages/ulauncher/main.py", line 20, in <module>
    from ulauncher.ui.windows.UlauncherWindow import UlauncherWindow
  File "/usr/lib/python2.7/site-packages/ulauncher/ui/windows/UlauncherWindow.py", line 38, in <module>
    from .PreferencesUlauncherDialog import PreferencesUlauncherDialog
  File "/usr/lib/python2.7/site-packages/ulauncher/ui/windows/PreferencesUlauncherDialog.py", line 12, in <module>
    from ulauncher.ui.AppIndicator import AppIndicator
  File "/usr/lib/python2.7/site-packages/ulauncher/ui/AppIndicator.py", line 4, in <module>
    gi.require_version('AppIndicator3', '0.1')
  File "/usr/lib/python2.7/site-packages/gi/__init__.py", line 129, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace AppIndicator3 not available
```

**Comment by xtraeme** [comment]
That means that `libappindicator` wasn't built with `gobject-introspection` support.

I'm testing the fix...

**Comment by paualberto** [comment]
Oh hey, wasn't expecting to see a comment from you boss.
Thanks for the fix! Hope you're doing well.

**Comment by xtraeme** [comment]
Thanks, I'm back, but not at full time :-)

**Comment by paualberto** [comment]
This was fixed in d46a32b (thanks @xtraeme !)
Template now works and has all dependencies.

**Comment by martnnw** [comment]
Ulauncher is in the repository now. This issue can be closed.
